lib/crazy_eight_game.rb

```diff
@@ -0,0 +1,161 @@
+require 'JSON'
+require 'rest-client'
+require 'pry'
+
+class CrazyEightGame < ActiveRecord::Base
+
+    def new_deck
+        self.deck_id = JSON.parse(RestClient.get("https://deckofcardsapi.com/api/deck/new/shuffle/"))['deck_id']
+        update_remaining(JSON.parse(RestClient.get("https://deckofcardsapi.com/api/deck/new/shuffle/"))['remaining'])
+        save
+    end
+
+    def draw_from_api(count)
+        response = RestClient.get("https://deckofcardsapi.com/api/deck/#{deck_id}/draw/?count=#{count}")
```

Comment:
```suggestion
        draw_json = JSON.parse(RestClient.get("https://deckofcardsapi.com/api/deck/#{deck_id}/draw/?count=#{count}"))
```

Not quite sure exactly what the response object is since I can't test, but once again the raw data isn't really useful to us. Storing the response was a good idea, but you should really store the JSON object, otherwise we will just have to parse it later.